### ISSUE TITLE
[intelliJ][Linux]Error message pops up when list clusters.

### ISSUE THREAD

**Issue by v-jiche** [Microsoft/azure-tools-for-java#2392]
Build:
artifacts/azure-toolkit-for-intellij-183.4284.118.PR.2528.11-21-2018.zip

Repro Steps:
1. Open Azure Explorer.
2. Choose HDInsight.

Result:
HDInsight shows blank. The error detail pops up.
![image](https://user-images.githubusercontent.com/35950097/48824681-e9a09b80-ed9f-11e8-8852-5716179cd3fb.png)
![image](https://user-images.githubusercontent.com/35950097/48824695-f9b87b00-ed9f-11e8-9feb-151f932438f8.png)
![image](https://user-images.githubusercontent.com/35950097/48824746-38e6cc00-eda0-11e8-8afb-75495f164a37.png)

java.util.NoSuchElementException: An error occurred while enumerating the result, check the original exception for details.
	at com.microsoft.azure.storage.core.LazySegmentedIterator.hasNext(LazySegmentedIterator.java:113)
	at java.util.Iterator.forEachRemaining(Iterator.java:115)
	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
	at com.microsoft.azure.hdinsight.serverexplore.hdinsightnode.StorageAccountNode.refreshItems(StorageAccountNode.java:106)
	at com.microsoft.tooling.msservices.serviceexplorer.RefreshableNode.refreshItems(RefreshableNode.java:96)
	at com.microsoft.tooling.msservices.serviceexplorer.RefreshableNode$2.run(RefreshableNode.java:148)
	at com.microsoft.intellij.helpers.IDEHelperImpl$2$1.run(IDEHelperImpl.java:157)
	at com.intellij.openapi.progress.impl.CoreProgressManager$TaskRunnable.run(CoreProgressManager.java:727)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$runProcess$2(CoreProgressManager.java:164)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:582)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:87)
	at com.intellij.openapi.progress.impl.CoreProgressManager.runProcess(CoreProgressManager.java:151)
	at com.intellij.openapi.progress.impl.CoreProgressManager$4.run(CoreProgressManager.java:403)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:314)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.microsoft.azure.storage.StorageException: Server failed to authenticate the request. Make sure the value of Authorization header is formed correctly including the signature.
	at com.microsoft.azure.storage.StorageException.translateException(StorageException.java:89)
	at com.microsoft.azure.storage.core.StorageRequest.materializeException(StorageRequest.java:315)
	at com.microsoft.azure.storage.core.ExecutionEngine.executeWithRetry(ExecutionEngine.java:175)
	at com.microsoft.azure.storage.core.LazySegmentedIterator.hasNext(LazySegmentedIterator.java:109)
	... 25 more

**Comment by v-jiche** [comment]
Bug repros in Build:
azure-toolkit-for-intellij-183.4284.118.develop.900.12-12-2018

**Comment by wezhang** [comment]
We can catch it here:
```
            getBlobContainers(connectionString).forEach(blobContainer -> {
                addChildNode(new BlobContainerNode(this, blobStorageAccount, blobContainer, !StringHelper.isNullOrWhiteSpace(defaultContainer) && defaultContainer.equals(blobContainer.getName())));
            });
```
And throw the root cause exception.

**Comment by t-rufang** [comment]
Fixed in #2575

**Comment by v-jiche** [comment]
Verified, Fixed.
Build: azure-toolkit-for-intellij-2018.3.develop.929.12-28-2018
